packages/metadata/src/metadata-service.ts

```diff
@@ -198,7 +198,7 @@ export class MetadataService implements IMetadataService {
     /**
      * Set map of uris from relative to absolute.
      *
-     * @param uriMap
+     * @param uriMap uri map
```

Comment:
This description does not add more value than the variable name. Perhaps can add what kind of mapping it is similar to what is in the methods description. It would be a bit repetitive, but at least it provides a bit more details about the parameter.

Reply:
updated